docs/concepts/alert_activations.ipynb

```diff
@@ -0,0 +1,542 @@
+   "id": "236497c2-e1eb-4f34-a8b5-a18c4f9969d7",
+   "metadata": {},
+   "source": "If the `entity` parameter is passed as a string without a tilde (`~`), the search is performed for an exact match with the given string:"
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 24,
+   "id": "3821679c-050c-448d-9d28-d73b0396786b",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "2"
+      ]
+     },
+     "execution_count": 24,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "activations = project.list_alert_activations(\n",
+    "    entity=\"test-func-handler.db80cba0c4be4ee9b86a09cf12a89991\"\n",
+    ")\n",
+    "len(activations)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "15c4642b-2fac-4bbf-b69d-7769e087240b",
+   "metadata": {},
+   "source": [
+    "## List alert activations for a given time range\n",
+    "\n",
+    "To filter, pass `since` and `until` as `datatime.datatime` objects:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 28,
+   "id": "e2a5713e-67a5-4ea9-a113-fd9ec699a7ae",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "0"
+      ]
+     },
+     "execution_count": 28,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import datetime\n",
+    "\n",
+    "activations = project.list_alert_activations(since=datetime.datetime.now())\n",
+    "len(activations)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 29,
+   "id": "f1cf11f6-6521-456c-b38a-0eb44b00d8c7",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "1"
+      ]
+     },
+     "execution_count": 29,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "activations = project.list_alert_activations(\n",
+    "    since=datetime.datetime(2024, 12, 11, 10, 5, 17, 674000),\n",
+    "    until=datetime.datetime.now(),\n",
+    ")\n",
+    "len(activations)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 30,
+   "id": "be0c33ac-a2a8-40fc-bebe-e36818fc9df7",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "4"
+      ]
+     },
+     "execution_count": 30,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "alert_config = run_db.list_alerts_configs()[0]\n",
+    "activations = alert_config.list_activations()\n",
+    "len(activations)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 34,
+   "id": "827f02f7-e60a-4a29-82d6-c818dae1a3b8",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "0"
+      ]
+     },
+     "execution_count": 34,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# update alert config and get activations since last update\n",
```

Comment:
Does it make sense to group updating the cfg and then viewing activations? Can an alert happen so quickly that you'll actually get results? Or did I misunderstand something?

Reply:
@jillnogold it is just an example. But in general, we might have an activation in that short period of time as well